src/postgres/event-handlers/CommentReactionCreated.ts

```diff
@@ -1,24 +1,30 @@
-import { CommentId } from '@subsocial/types/substrate/interfaces/subsocial';
-import { substrate } from '../../substrate/subscribe';
+import { PostId, Post } from '@subsocial/types/substrate/interfaces/subsocial';
 import { insertActivityForCommentReaction } from '../insert-activity';
 import { insertNotificationForOwner } from '../notifications';
-import { SubstrateEvent, EventHandlerFn } from '../../substrate/types';
+import { SubstrateEvent } from '../../substrate/types';
 
-export const onCommentReactionCreated: EventHandlerFn = async (eventAction: SubstrateEvent) => {
+export const onCommentReactionCreated = async (eventAction: SubstrateEvent, post: Post) => {
   const { data } = eventAction;
```

Comment:
Fix this copypasta on all post events handlers:

```typescript
const { data } = eventAction;
const follower = data[0].toString();
const commentId = data[1] as PostId;
```

it should be something like this everywhere:

```typescript
const { author, commentId } = parseCommentEvent(eventAction);
```